clients/CMakeLists.txt

```diff
@@ -116,13 +116,9 @@ add_custom_command(OUTPUT "${ROCSPARSE_GENTEST}"
 
 add_custom_target(rocsparse-common DEPENDS "${ROCSPARSE_COMMON}" "${ROCSPARSE_TEMPLATE}" "${ROCSPARSE_GENTEST}")
 
-# Client packaging
-include(../cmake/Packaging.cmake)
-
 if(BUILD_CLIENTS_SAMPLES OR BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
-  rocm_create_package_clients(LIB_NAME rocsparse
-                              DESCRIPTION "Radeon Open Compute SPARSE library"
-                              MAINTAINER "rocSPARSE Maintainer <rocsparse-maintainer@amd.com>"
-                              SECTION "dev"
-                              VERSION ${rocsparse_VERSION})
+        install(
+                DIRECTORY "${CMAKE_SOURCE_DIR}/build/release/clients/staging/"
```

Comment:
we can't hard code the 'release' directory

Reply:
use of CMAKE_BINARY_DIR instead ? :
"${CMAKE_BINARY_DIR}/clients/staging/"

Reply:
Or "${PROJECT_BINARY_DIR}/staging/*"  is what rocblas uses.  We  completely override rocm_create_package_clients not sure if the original never worked or who did the override.  See clients/CMakeLists.txt in rocblas for alternative example, @daineAMD any comment on this?

Reply:
Looks like Saad is the one the originally wrote client packaging (see 
https://github.com/ROCmSoftwarePlatform/rocBLAS/pull/602)

It looks to me like earlier in this file ${PROJECT_BINARY_DIR}/staging is used in the same sense as here (and in rocBLAS).